website/project/views/node.py

```diff
@@ -102,9 +103,11 @@ def project_new_post(auth, **kwargs):
 
     else:
         project = new_node('project', title, user, description)
+        new_project = _view_project(project, auth)
 
     return {
-        'projectUrl': project.url
+        'projectUrl': project.url,
+        'new_node': new_project['node'] if new_project else None
```

Comment:
For consistency with the other variables in the response, make `new_node` camelCase.